components/card-component/CardComponent.vue

```diff
@@ -15,12 +17,25 @@ export default {
   components: {
     SvgIcon
   },
-  props: [ 'body', 'count', 'image', 'path', 'slug', 'theme', 'title', 'video' ]
+  props: [ 'body', 'count', 'delay', 'image', 'path', 'slug', 'theme', 'title', 'video' ],
```

Comment:
Video and image aren't used

Reply:
I'll change this to a more orderly list of props!